path_data.go

```diff
@@ -370,8 +370,9 @@ func (b *versionedKVBackend) pathDataWrite() framework.OperationFunc {
 		}
 		if meta == nil {
 			meta = &KeyMetadata{
-				Key:      key,
-				Versions: map[uint64]*VersionMetadata{},
+				Key:           key,
+				Versions:      map[uint64]*VersionMetadata{},
+				LastOperation: string(req.Operation),
```

Comment:
Do we need this field anymore? We should  have the last operation as part of the Attribution.

Reply:
Don't think we do! I removed that field.